### DISCUSSION TITLE
Default timeout in Avocado instrumented test

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#5369] [Q&A] [answered]
Hi all,

Just to check what is the default timeout for a test or suit in nrunner? it looks like it is 24hrs but wasn't able to find the conf parameter related with it.

Is it possible to pass the timeout in another way more than specify it in the Avocado test class?

**Comment by marioaag** [comment]
Is there any suggestion about this and it is possible to set the timeout to infinite?

**Comment by richtja** [comment, marked as answer]
Hi @marioaag, I am so sorry for such a late reply. As you said, right now, we have a default timeout for instrumented tests set to 24 hours, you can find the declaration [here](https://github.com/avocado-framework/avocado/blob/master/avocado/plugins/runners/avocado_instrumented.py#L40). Unfortunately, right now there is no config parameter which could change this behavior.

We have open issues about timeouts which we have to deal with before we can solve this. But I created issue #5394 to describe your problem, so we will track this. 

Unfortunately we are working on different issues right now, so I can not tell when this will be solved, but if you have some spare cycles, the participation on those issues is welcome.
